coordinator/src/main/java/org/apache/uniffle/coordinator/AccessChecker.java

```diff
@@ -20,15 +20,20 @@
 import java.io.Closeable;
 
 /**
- *  Interface for checking the access info from the client-side.
+ *  Abstract class for checking the access info from the client-side.
  */
-public interface AccessChecker extends Closeable {
+public abstract class AccessChecker implements Closeable {
+  protected AccessManager accessManager;
```

Comment:
Why do we need a member `AccessManager` since we only need to use it to extract confs in the checker's constructor?

Reply:
Removed